src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/create_index_pattern_wizard.html

```diff
@@ -124,7 +128,10 @@ <h2 class="euiTitle euiTitle--small euiTextColor euiTextColor--subdued">
           ng-switch="controller.wizardStep"
         >
           <!-- Specify index pattern -->
-          <step-index-pattern
+          <div ng-switch-when="indexPattern">
+            <div id="stepIndexPatternReact"></div>
```

Comment:
Why not use `ng-react` to create a temporary directive to wrap the component? This way we'd be using a consistent mechanism for bridging angular to react. Here's an example of how I'm defining the [toasts directive wrapper](https://github.com/elastic/kibana/pull/15749/files#diff-2ed48bbde012eb7fb7a8fff695fbed06R127) and here's an example of [how it's consumed](https://github.com/elastic/kibana/pull/15749/files#diff-a73f844c42336db887feae3bfb7d52d7R17).

Reply:
I guess I didn't think it mattered since this file is going away in #15905

Reply:
OK your call